basyx/aas/backend/couchdb.py

```diff
@@ -244,7 +244,6 @@ def __init__(self, url: str, database: str):
         :param url: URL to the CouchDB
         :param database: Name of the Database inside the CouchDB
         """
-        super().__init__()
```

Comment:
Why did you remove these `super().__init__()` calls? I suspect they may have been unnecessary?

I guess if we would change the parent class, we'd have a look at all child classes as well, anyway so we would notice the missing `super().__init__()` call. On the other hand, does it hurt to have it here?

Reply:
I had probelms with the static-analysis check: "error: Call to abstract method "__init__" of "AbstractObjectStore" with trivial body via super() is unsafe  [safe-super]". I supposed, that we should remove these calls, if MyPy raises error

Reply:
Okay, I agree with this. 
Then it all LGTM